### ISSUE TITLE
Multiple lines in graph

### ISSUE THREAD

**Issue by Ami-Gohel** [margelo/react-native-graph#22]
This is not supporting multiple lines in a single chart.

**Comment by taatiq882** [comment]
Did you find any solution ? i also require multiple lime graph with sort of same implementation.

**Comment by chrispader** [comment]
This is not supported at the moment. Unfortunately, i don't think we plan to implement this feature anytime soon.

If you or somebody could open a PR for this, that would be great. What do you think @mrousavy ?

**Comment by mrousavy** [comment]
I think this requires a few structural changes and is a little bit of work. We don't plan to implement that anytime soon, but if you need this feature developed by us, contact us at hello@margelo.io :)

**Comment by sidferreira** [comment]
@mrousavy, would a PR be considered?

**Comment by Ami-Gohel** [comment]
I have creates multiple line graphs with SVG, soon I will make package and make open source.

**Comment by mrousavy** [comment]
@sidferreira 
> @mrousavy, would a PR be considered?

Yes! If you shoot me a PR, we can review it :)

**Comment by youssdevx** [comment]
@sidferreira a PR would be much appreciated 🙏🏽

**Comment by sidferreira** [comment]
@youssdevx although I had some local work, my main need was cancelled, so I don't know when I'll be able to

**Comment by youssdevx** [comment]
Could you maybe push your work? I'll try to work on it

**Comment by sidferreira** [comment]
I'll try

**Comment by sidferreira** [comment]
https://github.com/sidferreira/react-native-graph-tests

https://github.com/sidferreira/react-native-graph/tree/multiple-graphs

What I tried to do is to create a shared `const panGesture = usePanGesture({holdDuration: 100});` that is an argument for the graph, and then you can have a main graph and all others (using pointerEvents="none") will follow it

@youssdevx @mrousavy

**Comment by sidferreira** [comment]
I also tried to make the graph generic type

**Comment by youssdevx** [comment]
I'll look into it, many thanks 🙏🏽

**Comment by sidferreira** [comment]
@youssdevx https://github.com/margelo/react-native-graph/pull/66 too

**Comment by youssdevx** [comment]
I still did have the chance to work on this... @mrousavy do you have plans for this feature?

**Comment by youssdevx** [comment]
FYI, the new version of Victory has multiple lines, area charts...  => https://github.com/FormidableLabs/victory-native-xl

**Comment by erics1033** [comment]
Any updates on this feature request?

**Comment by youssdevx** [comment]
https://github.com/FormidableLabs/victory-native-xl 👀

**Comment by mrousavy** [comment]
> @mrousavy do you have plans for this feature?

No, I don't have the free time to build this for you guys. But if you submit a PR we can review it and merge it :)

**Comment by mihairadulescu** [comment]
Does anybody have a PR for this? with some draft ?

**Comment by SamuelLHuber** [comment]
we opened a PR now. Would love feedback on it to see how we can improve it tto fit the quality of this library.

 https://github.com/margelo/react-native-graph/pull/116
